test/features/data_qld_theme.feature

```diff
@@ -1,5 +1,18 @@
+@theme
 Feature: Theme customisations
 
+    @unauthenticated
+    Scenario: As a member of the public, when I go to the consistent asset URLs, I can see the asset
+        Given "Unauthenticated" as the persona
+        When I visit "/assets/css/main"
+        Then I should see "Bootstrap"
+        When I visit "/assets/css/font-awesome"
+        Then I should see "Font Awesome"
+        When I visit "/assets/css/select2"
+        Then I should see "select2-container"
+        When I visit "/assets/js/jquery"
+        Then I should see "jQuery"
```

Comment:
Should this also test "/assets/js/bootstrap"?

Reply:
Squiz doesn't seem to rely on it, and we don't need to supply a custom version of it, so I'm going with "not necessary".